### PR TITLE
fix: git error introduced in git 2.35.2

### PR DESCRIPTION
**Pull request by davefurmidge**
# Pull Request detail
Git v2.35.2 spec changes are causing this action to error during run.

`reviewdog: PullRequest needs 'git' command: failed to run 'git rev-parse --show-prefix': exit status 128`

## Related issues or PRs/関連issueやPR

https://github.com/reviewdog/action-misspell/issues/43
https://github.com/reviewdog/action-misspell/issues/42
https://github.com/reviewdog/reviewdog/issues/1158#issue-1202862728
https://github.com/reviewdog/action-yamllint/issues/18

close #.

## Fix or Add/Remove in this PR/このプルリクエストでやったこと

- ...

## Do not fix or add/remove this PR/このプルリクエストでやらなかったこと

- ...

## Pros and Cons/メリットとデメリット

- ...

## Test/動作確認

Test item list up with checkbox, checked after tested./チェックボックス式にし、確認後チェックする。

- [ ] ...

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -e
 if [ -n "${GITHUB_WORKSPACE}" ] ; then
   cd "${GITHUB_WORKSPACE}" || exit
 fi
-
+git config --global --add safe.directory $GITHUB_WORKSPACE || exit 1
```
Comment (tsuyoshicho) [inline entrypoint.sh:7]: Thanks, @davefurmidge san

And please change same as [actionlint fix](https://github.com/reviewdog/action-actionlint/pull/66)
(last line are space)

```bash
if [ -n "${GITHUB_WORKSPACE}" ] ; then
  cd "${GITHUB_WORKSPACE}" || exit
  git config --global --add safe.directory "${GITHUB_WORKSPACE}" || exit 1
fi

```

**Review by tsuyoshicho — CHANGES_REQUESTED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (tsuyoshicho) [reply entrypoint.sh:7]: I want to keep your contribution, so I want to fix it in this PR.

Comment (davefurmidge) [reply entrypoint.sh]: Thank you, changes made as requested

**Comment by tsuyoshicho** [comment]
@davefurmidge san,
Thank you for your contribution.

**Pull request merged**